crates/node_binding/src/js_values/mod.rs

```diff
@@ -1,5 +1,9 @@
+mod chunk {
+  // TODO: should we merge rspack_binding_options and node_binding?
```

Comment:
cc @h-a-n-a

Reply:
> or `rspack_plugin_xxx` export `RawXxxOptions` under feature `node_binding`

This looks good to me, keeping these options in the same crate.

Reply:
Just for heads-up. @Boshen seems treat every undefined symbol to lookup at runtime. For convenience, you can omit the feature. But still, I would love to add one though.

Reply:
> or `rspack_plugin_xxx` export `RawXxxOptions` under feature `node_binding`

I'm not familiar with napi or cargo or some build stuff, not sure if it's possible or good for ci build time, so you guys can decide it

Reply:
Let's stick to feature, it's a common practice and do not coupled with linker options.